shared/test/test_regexp.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'test_helper'
+
+require 'cdo/regexp'
+class RegexpUtilsTest < Minitest::Test
+  EXPECTED_DIGITS = '1234567890'
+  VALID_US_PHONE_NUMBERS = %w[
```

Comment:
Oh, funny! Looks like there is another test file for this in https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/test/test_regexp_utils.rb#L7-L7 (think that was before shared existed, though)